charts/karmada/values.yaml

```diff
@@ -798,8 +798,7 @@ schedulerEstimator:
           XXXXXXXXXXXXXXXXXXXXXXXXXXX
           -----END RSA PRIVATE KEY-----
   ## @param schedulerEstimator.labels labels of the scheduler-estimator deployment
-  labels:
-    app: karmada-scheduler-estimator
+  labels: {}
```

Comment:
I may prefer keeping this label (`app: karmada-scheduler-estimator`) in `value.yaml`.

But change the key name of the label in `karmada-scheduler-estimator.yaml`, such as `targetCluster: {{ $clusterName}}`

Reply:
I used a more concise name: `cluster`, and this label already existed.

Reply:
okey, yours is better~ 👍